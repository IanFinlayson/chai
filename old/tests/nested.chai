def main():
    var maps = [{"a": 1, "b": 2}, {"c": 3, "d": 4, "e": 5}, {:}]
    maps[1]["f"] = 6
    print(maps[0]["b"])
    print(maps)
    
    var nums = {"ian": [26, 10, 84], "caitie": [9, 10, 84]}
    nums["ryan"] = [28, 3, 12]
    nums["caitie"] += [19]
    print(nums)

