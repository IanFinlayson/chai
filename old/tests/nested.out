2
[{"a": 1, "b": 2}, {"c": 3, "d": 4, "e": 5, "f": 6}, {}]
{"ryan": [28, 3, 12], "ian": [26, 10, 84], "caitie": [9, 10, 84, 19]}
